### PR TITLE
add support for disqusjs 1.3

### PR DESCRIPTION
**Pull request by enpitsuLin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/disqusjs/tslint.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false
```
Comment (sandersn) [inline types/disqusjs/tslint.json:4]: You should remove this rule exclusion.

There aren't any type parameters in the code, so it's not doing anything.

**Review by sandersn — CHANGES_REQUESTED** [review]

File: types/disqusjs/tslint.json (hunk 1 above)
Comment (enpitsuLin) [reply types/disqusjs/tslint.json:4]: Thank you for review, so can I just remove it, right?

Comment (sandersn) [reply types/disqusjs/tslint.json]: no, sorry, you still need the `extends` part =(

Comment (enpitsuLin) [reply types/disqusjs/tslint.json]: That's okay, this will let me know that I should better understand tslint and the contribution process, that's cool😁

**Review by sandersn — APPROVED** [review]

**Pull request merged**